openhands/server/session/session.py

```diff
@@ -45,6 +46,7 @@ def __init__(
             EventStreamSubscriber.SERVER, self.on_event
         )
         self.config = config
+        self.loop = asyncio.get_event_loop()
```

Comment:
CC @tofarr who is working on making our asyncio stuff better--not sure if this is the right direction or not

Reply:
Sonnet recommended that, but I'll have to retry if that is/was actually part of the solution, i.e. it might actually not be needed anymore.

Reply:
This is basically to make sure, that the callback runs on the same thread